frontend/src/hooks/useVerifyBoardEffect.ts

```diff
@@ -13,23 +14,29 @@ export interface useVerifyBoardStateInterface {
     | undefined
   >
   isLoading: boolean
+  isServerError: boolean
   errorMessage: string
 }
 
 export const useVerifyBoardEffect = (): useVerifyBoardStateInterface => {
   const [isLoading, setIsLoading] = useState(false)
+  const [isServerError, setIsServerError] = useState(false)
   const [errorMessage, setErrorMessage] = useState('')
 
   const verifyJira = async (params: BoardRequestDTO) => {
     setIsLoading(true)
     try {
       return await boardClient.getVerifyBoard(params)
     } catch (e) {
-      const err = e as Error
-      setErrorMessage(`${params.type} ${VERIFY_FAILED_ERROR_MESSAGE}: ${err.message}`)
-      setTimeout(() => {
-        setErrorMessage('')
-      }, ERROR_MESSAGE_TIME_DURATION)
+      const err = e as AxiosError
```

Comment:
Duplicated code.
Refactor verifyJira.... to a function like `apiCall(clientMethod, errorMessage)`, use it in all `use...Effect`